products/docker-engine.md

```diff
@@ -20,6 +20,13 @@ auto:
     regex: ^v(?<major>0|[1-9]\d*)\.(?<minor>\d*)\.(?<patch>0|[1-9]\d*)(-ce)?$
 
 releases:
+
+-   releaseCycle: "24.0"
```

Comment:
Following https://github.com/moby/moby/discussions/45104, the doc regarding support has been updated to :

> Docker Engine is an open source project, supported by the Moby project maintainers and community members. Docker doesn’t provide support for Docker Engine. Docker provides support for Docker products, including Docker Desktop, which uses Docker Engine as one of its components.
> 
> For information about the open source project, refer to the [Moby project website](https://mobyproject.org/).

Don't know how to interpret this... Should we switch to the following new model to guess EOL date : eol(x) = releaseDate(x + 1) ?

Reply:
A dedicated issue has been opened to tackle this : #3006.